python/setup.py

```diff
@@ -30,24 +31,59 @@
 
 cuda_include_dir = os.path.join(CUDA_HOME, "include")
 
-extensions = [
-    Extension(
-        "*",
-        sources=cython_files,
-        include_dirs=[
-            "../include/rmm",
-            "../include",
-            "../build/include",
-            os.path.dirname(sysconfig.get_path("include")),
-            cuda_include_dir,
-        ],
-        library_dirs=[get_python_lib(), os.path.join(os.sys.prefix, "lib")],
-        libraries=["rmm"],
-        language="c++",
-        extra_compile_args=["-std=c++14"],
-    )
+try:
+    nthreads = int(os.environ.get("PARALLEL_LEVEL", "0") or "0")
+except Exception:
+    nthreads = 0
+
+include_dirs = [
+    "../include/rmm",
+    "../include",
+    "../build/include",
+    os.path.dirname(sysconfig.get_path("include")),
+    cuda_include_dir,
 ]
 
+library_dirs = [get_python_lib(), os.path.join(os.sys.prefix, "lib")]
+
+# lib:
+extensions = cythonize(
```

Comment:
Looks like you're still calling `cythonize` on `extensions` in the actual `setup` call, I think that needs to be removed?

Reply:
Good catch - fixed.